data/hai ships.txt

```diff
@@ -87,7 +87,7 @@ ship "Flea"
 	engine 0 27
 	gun 0 -27 "Railgun"
 	explode "tiny explosion" 15
-	description `This is a combat drone designed to help contain Unfettered aggression with a minimum risk to Hai life.`
+	description `The Flea is a light combat ship designed to help contain Unfettered aggression, while keeping a minimum risk to Hai life using remote control rather than pilots.`
```

Comment:
Are they really RC though? Or do the Hai have drone AI?

```
The Flea is a light, combat-focused drone that the Hai use to combat the recent increase in aggression by the Unfettered. The lives of Hai pilots are safeguarded through the use of remote control drone technology.
```
This tech could come from a Wanderer-Hai collaboration after the Wanderers obtain the Kor Sestor control receivers...